index.html

```diff
@@ -1761,11 +1741,31 @@
             "PresentationRequest" data-lt="start">select a presentation
             display</a>, the <a>user agent</a> MUST <dfn>monitor the list of
             available presentation displays</dfn> by running the following
-            steps.
+            steps:
           </p>
           <ol link-for="PresentationAvailability">
-            <li>Set the <a>list of available presentation displays</a> to the
-            empty list.
+            <li>Let <var>availabilitySet</var> be a shallow copy of the <a>set
+            of presentation availability objects</a>.
+            </li>
+            <li>If there is a pending request to <a for="PresentationRequest"
+            data-lt="start">select a presentation display</a> for a
+            <a>PresentationRequest</a> and if the <a>PresentationRequest</a>'s
+            <a>presentation display availability</a> is <code>null</code>, then
+            run the following substeps:
+              <ol>
+                <li>Let <var>A</var> be a newly created
```

Comment:
> Or are we assuming that all state mentioned in the spec is associated with a specific browsing context?

That would be my assumption unless stated otherwise (for instance, the definition of the "set of controlled presentations" is explicit that the set spans browsing contexts).

Aggregating the monitoring steps across browsing contexts is a valid optimization, but do we need to specify it? Does that change any visible behavior?

We could complete the note next to the algorithm to mention that user agents may want to handle monitoring across browsing contexts.

> > (By the way, shouldn't the above mentioned step 4 be extended to reject the promise if there is an unsettled promise in nested contexts or in ancestor contexts as well?)
> Good point and I likely agree, although it adds implementation complexity. File as a separate issue and I'll think about it.

Done in https://github.com/w3c/presentation-api/issues/394